examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
```

Comment:
What I'd like would be an example that shows Catalog standalone as shown here, but also embedded in a React Router app. Example off the top of my head: would it work this way?

```
ReactDOM.render(
  <Router …>
    <Route … />
    <Route component={<Catalog … />} />
  </Router>,
  document.getElementById('app')
);
```